front/components/data_source/WebsiteConfiguration.tsx

```diff
@@ -266,8 +267,8 @@ export default function WebsiteConfiguration({
           title="Add a Website"
           onSave={() => {
             setIsSubmitted(true);
-            validateForm();
-            if (isValid && !isSaving) {
+            const formValid = validateForm();
```

Comment:
Now do we still need the `isValid` state? Also, it seems we're calling again validateForm inside handleCreate, could we simplify?